client/src/views/files/FoldersPage.vue

```diff
@@ -689,7 +703,7 @@ async function openFileContextMenu(event: Event, file: parsec.EntryStat, onFinis
 }
 
 async function resetSelection(): Promise<void> {
-  fileItemRefs.value = [];
+  fileListItemRefs.value = [];
```

Comment:
Probably a mistake to clear this list. I even think this function is useless. It's just equivalent of calling `selectAllFiles(false)`

Reply:
After reflexion, we don't need the `resetSelection()` function. In future, selected files will remain selected while the user chooses the card or list view.